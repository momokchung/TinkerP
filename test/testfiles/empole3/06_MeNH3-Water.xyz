    11 92_MeNH3-Water_1.00
     1  N     -0.635097   -0.241601    0.273162    35     3     4     2     5
     2  H     -0.701594   -1.189162    0.641236    36     1
     3  H     -1.171272   -0.218097   -0.592711    36     1
     4  H      0.365437   -0.049953    0.043146    36     1
     5  C     -1.152753    0.758203    1.257002    37     1     6     7     8
     6  H     -1.060655    1.744724    0.818995    38     5
     7  H     -2.190130    0.536315    1.477240    38     5
     8  H     -0.551778    0.692606    2.155936    38     5
     9  O      1.989162    0.398945   -0.221685    16    10    11
    10  H      2.334509    0.946046   -0.933424    17     9
    11  H      2.770717    0.044319    0.212559    17     9
