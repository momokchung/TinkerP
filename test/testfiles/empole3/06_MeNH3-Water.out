
     ######################################################################
   ##########################################################################
  ###                                                                      ###
 ###            Tinker  ---  Software Tools for Molecular Design            ###
 ##                                                                          ##
 ##                      Version 8.10.5   February 2023                      ##
 ##                                                                          ##
 ##               Copyright (c)  Jay William Ponder  1990-2023               ##
 ###                           All Rights Reserved                          ###
  ###                                                                      ###
   ##########################################################################
     ######################################################################


 Intermolecular Energy :             -21.19421082 Kcal/mole

 Total Potential Energy :            -13.85099221 Kcal/mole

 Energy Component Breakdown :           Kcal/mole        Interactions

 Atomic Multipoles                   -13.85099221               33
